models/topics.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection.js');
+const format = require('pg-format');
+
+exports.fetchTopics = (path) => {
```

Comment:
Instead of passing down the path all the way here, please use `app.all('/*',()=>{})` to catch any bad paths.